packages/engine/src/proximityChecker/components/ProximityCheckerComponent.ts

```diff
@@ -0,0 +1,11 @@
+import { createMappedComponent, hasComponent, addComponent, getComponent } from '../../ecs/functions/EntityFunctions'
+
+export type ProximityCheckerComponentType = {}
+
+export const createProximityCheckerComponent = (eid: number) => {
```

Comment:
checking for a component should be done where this function is called, instead of in a function like this to improve readability of ecs logic